docs/sources/configuration/_index.md

```diff
@@ -2612,6 +2612,10 @@ shard_streams:
 
 # Minimum number of label matchers a query should contain.
 [minimum_labels_number: <int>]
+
+# The shard size defines how many index gateways should be used for querying.
```

Comment:
this description is a little misleading; if RF=5 and shard-size=1 I'd be using 5 gateways for querying, wouldn't I?

Reply:
ohh that makes sense.